src/app/static/src/tests/apiService.test.ts

```diff
@@ -279,9 +282,42 @@ describe("ApiService", () => {
         expect(warnings[1][0]).toBe("No success handler registered for request /baseline/.");
     });
 
+    it("passes language header on get", async () => {
```

Comment:
Testing that we're passing the `Accept-Language` header but can't write an integration test yet because the backend isn't there.